tla-io/src/main/scala/at/forsyte/apalache/io/typecheck/parser/DefaultType1Parser.scala

```diff
@@ -259,6 +264,11 @@ object DefaultType1Parser extends Parsers with Type1Parser {
     }
   }
 
+  // a reference to an alias
+  private def aliasReference: Parser[TlaType1] = {
+    DOLLAR() ~> newAlias ^^ { n => n }
```

Comment:
I believe `_ ^^ { n => n}`` is equivalent to `_.map(x => x)` in the context of the parser combinators, which means it's a no-op, and should be equivalent to:

```suggestion
    DOLLAR() ~> newAlias
```